kubeflow/optimizer/types/optimization_types_test.py

```
@@ -0,0 +1,254 @@
+        ),
+    ],
+)
+def test_objective(test_case: TestCase):
+    """Test Objective dataclass construction and direction coercion."""
+    print("Executing test:", test_case.name)
+    try:
+        obj = Objective(**test_case.config) if test_case.config else Objective()
+        assert test_case.expected_status == SUCCESS
+        assert obj.metric == test_case.expected_output["metric"]
+        assert obj.direction == test_case.expected_output["direction"]
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)